frontend/src/pages/profileSettings/ProfileSettingsView.jsx

```diff
@@ -0,0 +1,141 @@
+import EditIcon from '@mui/icons-material/Edit'
+import Footer from '../../components/layout/footer/FooterController'
+import HeaderCustom from '../../components/layout/headercustom/HeaderCustomController'
+import Logo from '../../images/logo.png'
+import classes from './profilesettings.module.scss'
+
+const ProfileSettingsView = ({
+    user,
+    updateProfile,
+    avatarOpen,
+    handleAvatarOpen,
+    handleAvatarClose,
+}) => (
+    <div>
+        <div style={{ flex: 'true' }}>
+            <HeaderCustom title="Edit Profile" />
+        </div>
+        <div className={classes.banner}>
+            <img className={classes.bannerImg} src={Logo} alt="Banner" />
+            <Button
+                className={classes.edit}
+                variant="contained"
+                endIcon={<EditIcon />}
+            >
+                Change Banner Image
+            </Button>
+        </div>
+
+        <Divider className={classes.divider} variant="middle" />
+
+        <div className={classes.flex}>
+            <img className={classes.avatar} src={Logo} alt="Avatar" />
+            <Button
+                variant="outlined"
+                endIcon={<EditIcon />}
+                onClick={handleAvatarOpen}
+            >
+                Edit Avatar
+            </Button>
+        </div>
+
+        <Divider className={classes.divider} variant="middle" />
+
+        <div className={classes.flex}>
+            <div className={classes.left}>
+                <p style={{ fontWeight: 'bold' }}>Username</p>
+            </div>
+            <div className={classes.right}>
+                <p style={{ color: 'grey' }}>{user.username}</p>
+            </div>
+        </div>
+
+        <Divider className={classes.divider} variant="middle" />
+        <div className={classes.flex}>
+            <div className={classes.left}>
+                <p style={{ fontWeight: 'bold' }}>Handle</p>
+            </div>
+            <div className={classes.right}>
+                <p style={{ color: 'grey' }}>{user.nickname}</p>
+            </div>
+        </div>
+        <p className={classes.subtext}>Your handle will appear as:</p>
+        <p className={classes.subtext} style={{ marginBottom: '14px' }}>
+            @{user.nickname}
+        </p>
+
+        <Divider className={classes.divider} variant="middle" />
+
+        <div className={classes.flex}>
+            <div className={classes.left}>
+                <p style={{ fontWeight: 'bold' }}>Bio</p>
+            </div>
+            <div className={classes.right}>
+                <Button variant="text">Edit</Button>
+            </div>
+        </div>
+        <p className={classes.subtext} style={{ marginBottom: '14px' }}>
+            {user.bio}
+        </p>
+
+        <Divider className={classes.divider} variant="middle" />
+
+        <div className={classes.save}>
+            <Button type="submit" variant="contained" onClick={updateProfile()}>
```

Comment:
updateProfile() will be called everytime the view is rendered, you should remove the parenthesis

Reply:
done